service/src/main/java/com/epam/repair/service/RepairOrderServiceImpl.java

```diff
@@ -51,11 +50,11 @@ public RepairOrder update(RepairOrder repairOrder) {
     }
 
     @Override
-    public void deleteById(Integer repairOrderId) {
+    public void deleteById (Integer repairOrderId) {
         if (repairOrderDao.existsById(repairOrderId)) {
             repairOrderDao.deleteById(repairOrderId);
```

Comment:
What if somewhere between these operations connection to DB will be lost? Or another thread do the same operation.